f5_openstack_agent/lbaasv2/drivers/bigip/network_service.py

```diff
@@ -120,11 +120,46 @@ def initialize_tunneling(self):
                            vtep_selfip_name))
         return local_ips
 
+    def is_service_connected(self, service):
+        networks = service.get('networks', {})
+
+        for (network_id, network) in networks.iteritems():
+            if network_id in self.conf.common_network_ids:
+                continue
+
+            network_type = \
+                network.get('provider:network_type', "")
+            if network_type == "flat":
+                continue
+
+            segmentation_id = \
+                network.get('provider:segmentation_id', 0)
+            if not segmentation_id:
+                if network_type == 'opflex' or \
```

Comment:
if not seg_id and disconnected_enabled:
    if network_type in [supported_encapsulations]:
          return False